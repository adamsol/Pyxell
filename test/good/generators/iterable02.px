
func* range(a, b, step = 1): Rat def
    yield a...b by step

func* f(a: [Int]): Float def
    yield ...range(0, a.length, 2)
    yield 4.2
    yield ...a by -1

print [...f([5, 6, 3])]
