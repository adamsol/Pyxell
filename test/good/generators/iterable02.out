[0, 2, 4.2, 3, 6, 5]
